otherlibs/threads/stdlib.ml

```diff
@@ -306,6 +309,13 @@ let rec ( @ ) l1 l2 =
     [] -> l2
   | hd :: tl -> hd :: (tl @ l2)
 
+let list_rev l =
+  let rec rev_append l1 l2 = match l1 with
+    | [] -> l2
+    | x :: tail1 -> rev_append tail1 (x :: l2)
+  in
+  rev_append l []
```

Comment:
Can't invocations of this be replaced with `List.rev_append l []`?

Reply:
It's the same issue as having to redefine `protect` instead of using `Fun.protect`. I don't know exactly the internals, but afaik `Stdlib` cannot refer to the other modules.